pkgs/by-name/hy/hydrus/package.nix

```diff
@@ -142,6 +143,11 @@ python3Packages.buildPythonApplication rec {
     runHook postCheck
   '';
 
+  __darwinAllowLocalNetworking = true;
```

Comment:
Not sure if we then should set this already. Maybe it is not even the right fix?